scripts/zulip_emoji_merge_delegate.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import sys
+import zulip
+import requests
+import re
+
+# Usage:
+# python scripts/zulip_emoji_merge_delegate.py $ZULIP_API_KEY $ZULIP_EMAIL $ZULIP_SITE $GITHUB_TOKEN
+
+ZULIP_API_KEY = sys.argv[1]
+ZULIP_EMAIL = sys.argv[2]
+ZULIP_SITE = sys.argv[3]
+GITHUB_TOKEN = sys.argv[4]
+
+# Initialize Zulip client
+client = zulip.Client(
+    email=ZULIP_EMAIL,
+    api_key=ZULIP_API_KEY,
+    site=ZULIP_SITE
+)
+
+# Fetch messages from the past 1h15
+response = client.get_messages({
+    "anchor": "newest",
+    "num_before": 200,
+    "num_after": 0,
+    "narrow": [{"operator": "channel", "operand": "PR reviews"}],
+})
+
+messages = response['messages']
+pr_pattern = re.compile(r'https://github\.com/leanprover-community/mathlib4/pull/(\d+)')
+
+for message in messages:
+    content = message['content']
+    match = pr_pattern.search(content)
+    if match:
+        pr_number = match.group(1)
+        # Check for emoji reactions
+        reactions = message['reactions']
+        has_merge = any(reaction['emoji_name'] == 'merge' for reaction in reactions)
+        has_peace_sign = any(reaction['emoji_name'] == 'peace_sign' for reaction in reactions)
+
+        if not has_merge and not has_peace_sign:
```

Comment:
This means that the bot will only react to messages that do not already have a reaction.  Would it be better to have the bot react *always*?  After all, when a maintainer merges a PR, they usually react with :merge: when they `bors merge` the PR, not when the PR is actually merged.  The bot reaction would be confirmation that the PR really got merged after all.

Reply:
actually, maybe I should check for the `ready-to-merge` label, instead of `[Merged by bors]` in the title.

Reply:
Wait, that answers a different question

Reply:
Ok, that at least makes the two checks more consistent with one another!  :smile:

Reply:
I pushed an update that answers your question as well